warehousecontroller/mission2goal-translation-design.txt

```diff
@@ -0,0 +1,13 @@
+Mission Goal:
+	[TYPE, any/blue/green/red, ALL/[num], CURRENT/shelf1/shelf2/.., cbOut1/cbOut2/.]
+PDDL Goal:
+
+Num:CURRENT
+	Exists x1, .. , xn. AND_i ((type(xi) = T)(posn(xi) = cbOut2))
```

Comment:
If I am not mistaken, @eanevul, this is not the acceptable PDDL format and would need some further transformation before we can include it in the PDDL problem file?